elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -152,6 +152,40 @@ export const getUniquePeersbyIp = (
 	return [...peerMap.values()];
 };
 
+export const removeCommonIPsFromLists = (
```

Comment:
This method is using height which is a concept that must be removed from the library. Please, read the description of the issue, it is stated there.